SysBot.Pokemon.YouTube/Helpers/YouTubeCommandsHelper.cs

```diff
@@ -0,0 +1,88 @@
+using PKHeX.Core;
+
+namespace SysBot.Pokemon.YouTube
+{
+    public static class YouTubeCommandsHelper
+    {
+        // Helper functions for commands
+        public static bool AddToWaitingList(string setstring, string display, string username, out string msg)
+        {
+            if (!YouTubeBot.Info.GetCanQueue())
+            {
+                msg = "Sorry, I am not currently accepting queue requests!";
+                return false;
+            }
+
+            var set = TwitchShowdownUtil.ConvertToShowdown(setstring);
```

Comment:
Should probably just name the class to showdown util and put it in SysBot.Pokemon at this point if we are planning to expand platforms